src/transactions/transferFeedUtils.ts

```diff
@@ -400,3 +426,53 @@ export function isTransferTransaction(
 ): tx is TransferItemFragment {
   return (tx as TransferItemFragment).address !== undefined
 }
+
+// Note: This hook is tested from src/transactions/feed/TransferFeedItem.test.ts
+function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTransfer) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(amount.tokenAddress)
+  const fcTransferDetails = useSelector(getCachedFiatConnectTransferSelector(transactionHash))
+  const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
+  const account = useMemo(
+    () =>
+      fcTransferDetails
+        ? cachedFiatAccountUses.find(
+            ({ fiatAccountId }) => fiatAccountId === fcTransferDetails.fiatAccountId
+          )
+        : undefined,
+    [cachedFiatAccountUses, fcTransferDetails]
+  )
+  if (!account || !fcTransferDetails) {
+    return
+  }
+
+  const fiatAmount = Number(fcTransferDetails.quote.fiatAmount)
+  const total = Number(amount.value)
+  const exchangeRate = String(fiatAmount / total)
+  const sign = fcTransferDetails.quote.transferType === TransferType.TransferOut ? -1 : 1
+  const localAmount: LocalAmount = {
+    value: new BigNumber(fiatAmount).multipliedBy(sign),
+    currencyCode: convertToLocalCurrency(fcTransferDetails.quote.fiatType),
+    exchangeRate,
+  }
+
+  let subtitle: string
+  switch (account.fiatAccountType) {
+    case FiatAccountType.BankAccount:
+      subtitle = t('feedItemFcTransferBankAccount')
+      break
+    case FiatAccountType.MobileMoney:
+      subtitle = t('feedItemFcTransferMobileMoney')
+      break
+    default:
+      Logger.debug(TAG, 'useFiatConnectTransferDisplayInfo received an unsupported FiatAccountType')
+      return
+  }
+
+  return {
+    title: t('feedItemFcTransferWithdraw', { crypto: fcTransferDetails.quote.cryptoType }),
```

Comment:
This currently hardcodes the title for withdraws, but we do in fact check the transfer type above to determine the sign of the amount displayed. Since this doesn't work for deposits (cash-ins) atm, should we log an error and return `null` if `fcTransferDetails.quote.transferType === TransferType.TransferIn`?

Reply:
I added the `transferType` as another check that will cause an early return with `null`, so now the code only supports transfers out.